.github/workflows/website.yml

```diff
@@ -0,0 +1,29 @@
+name: website
+on:
+  push:
+    branches:
+      - master
+permissions:
+  contents: none
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  FASTLY_API_TOKEN: ${{ secrets.FASTLY_API_TOKEN }}
```

Comment:
Out of curiosity, where did we get these from? If we've copy/pasted them from somewhere then they'll need rotating.

Reply:
They haven't been added to the repo yet.  I assume the new values haven't been created.